packages/rancher-monitoring/rancher-monitoring/generated-changes/patch/templates/prometheus/rules-1.14/etcd.yaml.patch

```diff
@@ -10,8 +10,8 @@
  apiVersion: monitoring.coreos.com/v1
  kind: PrometheusRule
  metadata:
-@@ -291,4 +292,5 @@
-         {{ toYaml .Values.defaultRules.additionalRuleLabels | nindent 8 }}
```

Comment:
Same consideration here, do we need to keep these default rule relabellings?

Reply:
See above: https://github.com/rancher/charts/pull/3930#discussion_r1611735165